modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/Walkthrough.path

```diff
@@ -15,11 +15,26 @@
 	<td>//*[@class='lfr-walkthrough-hotspot']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CLOSE_POPOVER</td>
+	<td>//*[contains(@class,'popover')]//button[*[name()='svg'][contains(@class,'icon-times')]]</td>
+	<td></td>
+</tr>
 <tr>
 	<td>POPOVER</td>
 	<td>//*[contains(@class,'popover') and contains(@class,'show')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>POPOVER_SAMPLE_CONTENT</td>
+	<td>//*[contains(@class,'popover-body')][//span[text()='Content ${key_index}'] and //br and //code[text()='Hello${key_index}']]</td>
```

Comment:
created a sample content path like this because in [SampleWalkthrough.js](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-components-sample-web/src/main/resources/META-INF/resources/js/SampleWalkthrough.js#L24) the content is generated with `content: '<span>Content 1</span><br/><code>Hello1</code>',` in the JSON. So I wanted to capture the tags in the check.

Reply:
Sounds good :+1: